shared/teams/new-team/index.js

```diff
@@ -6,6 +6,32 @@ import {globalColors, globalMargins, globalStyles} from '../../styles'
 
 import type {Props} from './'
 
+const validTeamname = (s: string): boolean => {
+  // This logic is copied from go/protocol/keybase1/extras.go.
+  if (s.length < 2 || s.length > 16) {
+    return false
+  }
+
+  return /^([a-zA-Z0-9][a-zA-Z0-9_]?)+$/.test(s)
+}
+
+const headerText = (errorText: string, name: string): string => {
+  if (errorText) {
+    return errorText
+  }
+
+  const i = name.indexOf('.')
```

Comment:
:sheep: Should this look for the last index of `.`, i.e. if creating `keybase.builds.linux` should it say `You're creating a subteam of keybase.builds.`?

Reply:
Done!